src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/BridgeConfiguration.cs

```diff
@@ -33,8 +33,8 @@ public class BridgeConfiguration
         private const string BridgeTcpPort_PropertyName = "BridgeTcpPort";
```

Comment:
I don't believe BridgeConfiguration is used by the new infrastructure for cert gen.  If true, I'd prefer not changing it, and we can nuke it on a follow-up PR.  If I'm wrong, ignore this comment.